srcpkgs/cross-arm-none-eabi-gdb/template

```diff
@@ -18,7 +18,7 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-3.0-or-later"
 homepage="https://www.gnu.org/software/gdb"
 distfiles="${GNU_SITE}/gdb/gdb-${version}.tar.xz"
-checksum=cccfcc407b20d343fb320d4a9a2110776dd3165118ffd41f4b1b162340333f94
+checksum=0e1793bf8f2b54d53f46dea84ccfd446f48f81b297b28c4f7fc017b818d69fed
 make_check=no  # See gdb/template.
 patch_args="-Np1"
```

Comment:
this line shouldn't be necessary now, could you remove it?

Reply:
Sure thing

Reply:
@paper42 I have removed the patch file as well

Reply:
It still applies, no? Is it fixed upstream in a different way?

Reply:
So actually I only made sure that it still builds without the patch in gcc12 and current gcc

Reply:
I would guess this is only required for mips and other gdb packages still have it, so if you are not sure it's not required anymore, please don't remove it.

cc @unspecd

Reply:
Ah Okay